### DISCUSSION TITLE
Tailwind css generated all class but don't works the media queries

### DISCUSSION THREAD

**Discussion by edw19** [tailwindlabs/tailwindcss#8977] [Help]
I have this configuration for generated all class

someone who can help me ?

module.exports = {
  safelist: [
    {
      pattern: /./, // the "." means "everything"
    },
  ],
  theme: {
    fontSize:{
      'xs':'.75rem',
      'sm': '.875rem',
      'tiny': '.875rem',
      'base': '1rem',
      'lg': '1.125rem',
      'xl': '1.25rem',
      '2xl': '1.5rem',
      '3xl': '1.875rem',
      '4xl': '2.25rem',
      '5xl': '3rem',
      '6xl': '4rem',
      '7xl': '5rem',
    },
    extend: {
      colors: {
        primary: {
          DEFAULT: 'rgb(var(--color-primary) / <alpha-value>)',
        },
        secondary: {
          DEFAULT: 'var(--color-secondary) / <alpha-value>',
          700: '#9747FF',
        },
        alternative: {
          DEFAULT: 'var(--color-alternative)'
        },
        error: {
          first:'var(--color-error)',
          middle:'#FF8C3E',
          alert:'#FFABAB'
        }
      }
    }
  },
  plugins: [
    require('flowbite/plugin')
  ],
}

**Comment by wongjn** [comment]
The regex in `safelist` is only matched against "base" classes, such as `text-lg` but **won't match against variants** such as `md:text-lg`. You need to specify the additional variants you would like generated using the `variants` key:
```js
module.exports = {
  safelist: [
    {
      pattern: /./,
      variants: [
        'lg',
        'md',
        // keep going if you want!
      ],
    },
  ],
  // …
};
```

See the [documentation](https://tailwindcss.com/docs/content-configuration#using-regular-expressions) for full details on this.
